.github/workflows/build.yml

```diff
@@ -5,7 +5,7 @@ on:
     # Sequence of patterns matched against refs/heads
     branches:    
       # Push events on main branch
-      - 'main'
+      - 'release/0.8.x'
```

Comment:
I would just add the release branch in addition to main, instead of replacing main. I suppose it doesn't matter...I am not really sure.